custom-words.txt

```diff
@@ -4003,4 +4003,41 @@ Opex
 opex
 decomission
 Decommisioned
-Decommision
\ No newline at end of file
+Decommision
+exadata
+adbscharsetname
+autonomousdbversionsname
+dbsystemshapename
+dnsprivateviewocid
+ocid
+dnsprivatezonename
+giversionname
+autonomousdatabasename
+cloudexadatainfrastructurename
+dbserverocid
+cloudvmclustername
+dbnodeocid
+virtualnetworkaddressname
+ocids
+ords
+ncharacter
+ecpu
+adbsncharsetname
+provisionable
+ocpu
+reco
+exaoverview
+diskgroup
+clusterware
+tcps
+vmbm
+dbcs
+iorm
+tpurgent
+deregistering
+vmdb
+vnic
+exacc
+ezconnect
+ezconnectplus
+dataguard
```

Comment:
A lot of these look like they are intended to be acronyms, or camelCase, maybe they should not really be being added as all lower case?
[Also to prevent other RPs using them as typos accidentally it might be better to have them in cSpell, if they're not going to be significantly reusable?]